WordPress/src/main/java/org/wordpress/android/ui/plugins/PluginDetailActivity.java

```diff
@@ -329,8 +337,9 @@ public void onClick(DialogInterface dialogInterface, int i) {
 
     private void showDomainRegistrationDialog() {
         DialogFragment dialogFragment = new DomainRegistrationPromptDialog();
-        dialogFragment.show(getSupportFragmentManager(),
-                DomainRegistrationPromptDialog.DOMAIN_REGISTRATION_PROMPT_DIALOG_TAG);
+        FragmentTransaction ft = getSupportFragmentManager().beginTransaction();
+        ft.add(dialogFragment, DomainRegistrationPromptDialog.DOMAIN_REGISTRATION_PROMPT_DIALOG_TAG);
+        ft.commitAllowingStateLoss();
```

Comment:
What was the reasoning behind this change? I feel like `commitAllowingStateLoss` should be used as a last resort.

Reply:
ok looking at it again, I think I understand the reason. If the events arrives, after the app went to the background the app would crash. I can't think of any better solution without sticky events and this dialog isn't critical anyway, so I guess it's fine.